recipes/tidehunter/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
     - {{ compiler('c') }}
   host:
     - zlib
+  run:
+    - zlib
```

Comment:
This makes no sense

Reply:
zlib can be deleted here ... zlib has a run-dependency defined and it is not be needed in the run-section.